source/js/localization.js.es6

```diff
@@ -108,6 +108,8 @@ const localization = {
       ].map(path => Array.from(document.querySelectorAll(`a[href^="${path}"]`)))
     ).forEach((element) => {
       if (element.hasAttribute('data-no-localize')) return; // don't localize urls with no-localize
+      if (!urlHelper.shouldLocalizeUrl(element.href, localeId)) return; // don't localize
```

Comment:
As noted in `shouldLocalizeUrl`, this would always return falsey, so I'd assume this is broken.

Fairly certain we have no test coverage for this stuff, so be wary!